### PR TITLE
Clean make tidy

### PR DESCRIPTION
**Pull request by mrKisaoLamb**

### CODE REVIEW

**Comment by gibizer** [comment]
/test nova-operator-build-deploy
Ceph issue?
```
pod/ceph created
+ '[' 0 -lt 15 ']'
bash scripts/gen-ceph-kustomize.sh "isready"

~/install_yamls/out/openstack/ceph/cr ~/install_yamls
Waiting the cluster to be up
..........................................................................................make: *** [Makefile:1174: ceph] Error 1
```

File: Makefile [hunk 1]
```diff
@@ -120,13 +120,10 @@ vet: gowork ## Run go vet against code.
...
 
-APIPATH ?= $(shell pwd)/api
 .PHONY: tidy
 tidy: ## Run go mod tidy on every mod file in the repo
-	go mod tidy; \
-	pushd $(APIPATH); \
-	go mod tidy; \
-	popd
+	go mod tidy
+	cd ./api & go mod tidy
```
Comment (gibizer) [inline Makefile:126]: Somehow it does not work as expected:
```
$ git log --oneline | head -n1
e0c9624 Clean make tidy
$ rm go.sum api/go.sum 
$ make tidy
go mod tidy
cd ./api & go mod tidy
$ git status
On branch pr-331-make-tidy
Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	deleted:    api/go.sum

no changes added to commit (use "git add" and/or "git commit -a")
```

Comment (gibizer) [reply Makefile:126]: //later
I see it now. `&` vs `&&`

**Review by gibizer — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (mrKisaoLamb) [reply Makefile]: Thanks! Fixed now

**Review by gibizer — APPROVED** [review]
looks good

**Pull request merged**